docs/gettingstarted/structuring.md

```diff
@@ -1,124 +1,81 @@
+This is even more relevant when it comes to loading modules. If there are class files in two packages under the same name in separate modules, this will cause the mod loader to crash on startup since mod modules are exported to the game and other mods.
 
-|     Property  |   Type   | Default           | Description |
-|-------------: |:--------:|:--------:         |:------------|
-|        modid  |  string  | mandatory         | The modid this file is linked to. |
-|      version  |  string  | mandatory         | The version of the mod. It should be just numbers separated by dots, ideally conforming to Forge's [Semantic Versioning][versioning] structure. |
-|  displayName  |  string  | mandatory         | The user-friendly name of this mod. |
-| updateJSONURL |  string  |   `""`            | The URL to a [version JSON][updatechecker]. |
-|   displayURL  |  string  |   `""`            | A link to the mod's homepage. |
-|     logoFile  |  string  |   `""`            | The filename of the mod's logo. It must be placed in the root resource folder, not in a subfolder. |
-|      credits  |  string  |   `""`            | A string that contains any acknowledgements you want to mention. |
-|      authors  |  string  |   `""`            | The authors of this mod. |
-|  description  |  string  | mandatory         | A description of this mod. |
-| displayTest   | string   | `"MATCH_VERSION"` | Controls the display of the mod in the server connection screen.
-| dependencies  | [list]   |   `[]`            | A list of dependencies of this mod. |
+```
+module A
+  - package X
+    - class I
+    - class J
+module B
+  - package X // This package will not normally be loaded, preventing usage of R, S, T
+    - class R
+    - class S
+    - class T
+```
 
-\* All version ranges use the [Maven Version Range Specification][mvr].
+As such, your top level package should be something that you own: a domain, email address, a subdomain of where your website, etc. It can even be your name or username as long as you can guarantee that it will be uniquely identifiable within the expected target.
 
-The Mod File
-------------
+Type      | Value             | Top-Level Package
+:---:     | :---:             | :---
+Domain    | example.com       | `com.example`
+Subdomain | example.github.io | `io.github.example`
+Email     | example@gmail.com | `com.gmail.example`
 
-Generally, we will start with a file named after your mod and put into your package. This is the *entry point* to your mod and will contain some special indicators marking it as such.
+The next level package should then be your mod's id (e.g. `com.example.examplemod` where `examplemod` is the mod id). This will guarantee that, unless you have two mods with the same mod id (which should never be the case), your packages should not have any issues loading.
```

Comment:
```suggestion
The next level package should then be your mod's id (e.g. `com.example.examplemod` where `examplemod` is the mod id). This will guarantee that, unless you have two mods with the same id (which should never be the case), your packages should not have any issues loading.
```